protos/pbft_message.proto

```diff
@@ -52,35 +52,34 @@ message PbftMessage {
   PbftBlock block = 2;
 }
 
-
-// View change message, for when a node suspects the primary node is faulty
-message PbftViewChange {
+// A message sent by the new primary to signify that the new view should be
+// started
+message PbftNewView {
   // Message information
   PbftMessageInfo info = 1;
 
-  // Consensus seal proving the correctness of the block committed at the
-  // sequence number specified in the message info
-  PbftSeal seal = 2;
+  // 2f ViewChange messages
```

Comment:
For clarity, maybe "ViewChange messages, of length 2f" to make it clearer that '2f' is in reference to the length.